Makefile

```diff
@@ -312,6 +311,11 @@ load-images: kubectl kind k3d docker-build ## Push docker images to the local ki
 	@if [ $$($(KUBECTL) config current-context) = k3d-$(CLUSTER_NAME) ]; then make k3d-load;  fi
 	@if [ $$($(KUBECTL) config current-context) = kind-$(CLUSTER_NAME) ];  then make kind-load; fi
 
+.PHONY: load-test-vm
```

Comment:
That's `example-vms`, but without it's dependencies?

In that case, can we have `load-example-vms` with image import, and then `example-vms: docker-build-examples kind k3d kubectl local-example-vms`, without the body?

Reply:
And I think rename anyway should be done, better to have names consistent.

Reply:
Makes sense — renamed and changed the other targets to match in 49066ac448f15f8471c537c39e4a398ebd71c4a9. wdyt?